index.html

```diff
@@ -1,19 +1,84 @@
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" href="./css/normalize.css">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="./css/common.css">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
-<body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+<body>
+    <header class="align-center flex">
+        <img src="./img/karma-logo.svg" alt="Karma Logo" />
+        <nav>
+            <ul class="flex">
+                <li>
+                    <a href="./store.html" class="active-link">Meet Karma</a>
+                </li>
+                <li>
+                    <a href="#">How it Works</a>
+                </li>
+                <li>
+                    <a href="./store.html">Store</a>
+                </li>
+                <li>
+                    <a href="#">Blog</a>
+                </li>
+                <li>
+                    <a href="#">Help</a>
+                </li>
+                <li>
+                    <a href="#">Login</a>
+                </li>
+            </ul>
+        </nav>
+        <img src="img/menu-hamburger.svg" class="hamburger-menu">
 
+    </header>
+    <main>
+        <section class="hero justify-center align-center flex">
+            <h1 class="font-weight">Introducing Karma</h1>
+            <p class="font-weight">Bring WiFi with you, everywhere you go.</p>
+            <button>Learn More</button>
+        </section>
+        <section class="karma-details text-center">
+            <h2 class="font-weight">Everyone needs a little Karma</h2>
+            <div class="row-container flex">
+                <div class="column-container align-center flex">
+                    <img src="./img/icon-devices.svg" alt="Image outline of devices that can connect to the internet" />
+                    <p>Internet for all devices</p>
+                </div>
+                <div class="column-container align-center flex">
+                    <img src="./img/icon-coffee.svg" alt="Image outline of a steaming cup of coffee" />
+                    <p>Boost your productivity</p>
+                </div>
+                <div class="column-container align-center flex">
+                    <img src="./img/icon-refill.svg" alt="Image outline of a gas pump" />
+                    <p>Pay as You Go</p>
+                </div>
+            </div>
+        </section>
+        <section class="feature align-center text-center">
+            <img src="./additional/feature-img.png" />
+            <div class="feature-text-wrapper justify-center align-center flex">
+                <p class="font-weight quotes"><em>Wherever I am, I just don't worry about my connection anymore! </em></p>
+                <button>Get Karma Today</button>
```

Comment:
In the assingment, we were advised to make this button clickable. 
I would enclose the button in a form and add action like so: <form action="./store.html">

Reply:
I'll change that now